src/gascheduler.erl

```diff
@@ -35,7 +35,7 @@
 %% Types
 -type worker_nodes()  :: [node()].
 -type max_workers()   :: non_neg_integer().
--type max_retries()   :: pos_integer().
+-type max_retries()   :: pos_integer() | infinity.
```

Comment:
In line 356  we should also use `max_retries()` type instead of `non_neg_integer() | infinity`

Reply:
@lastres good idea!